app/views.py

```diff
@@ -1,9 +1,23 @@
-from flask import Blueprint, render_template
-
+from flask import Blueprint, render_template, request
+from . import database as db
 views = Blueprint('views', __name__, template_folder='templates')
+import json
+
 
 @views.route('/skema', methods=['GET'])
 def main_page():
-    return render_template('skema.html')
+    subjects = db.get_subject()
+    profs = db.get_profs()
+    return render_template('skema.html', subjects=subjects, profs=profs)
 
 
+@views.route('/data')
+def return_data():
+    requestArgs = request.args
+    start_date = request.args.get('start', '')
+    end_date = request.args.get('end', '')
+    # You'd normally use the variables above to limit the data returned
+    # you don't want to return ALL events like in this code
+    # but since no db or any real storage is implemented I'm just
+    # returning data from a text file that contains json elements
```

Comment:
Do we need all this?  I think we can remove everything except the return at the requestArgs